packages/transformer/src/IModelTransformer.ts

```diff
@@ -180,6 +180,15 @@ export interface IModelTransformOptions {
    * @default false
    */
   ignoreMissingChangesetsInSynchronizations?: boolean;
+
+  /** If 'true', try to handle conflicting code value changes by using temporary guid values.
+   * This solves an issue when an element update fails because of a duplicate Element CodeValue in cases where the duplicate target element is set to be deleted in the future,
+   * or when elements are switching code values with one another and we need a temporary code value.
+   * Code values will be resolved to their intended values during [[finalizeTransformation]].
+   *
+   * @default false
+   */
+  handleElementCodeDuplicates?: boolean;
```

Comment:
why put it behind an option? I don't remember if I suggested that... the current implementation doesn't seem like it needs that.

Reply:
removed the option